SparkPost/security-badger
# FE-1512 - Update vulnerable dependencies in security-badger

[FE-1512](https://sparkpost.atlassian.net/browse/FE-1512)

### What Changed

- Updates `eslint`, `eslint-plugin-import` and `jest` to resolve vulnerable dependencies
- Reinstalls some other packages to automatically consume patches (see changes to `package-lock.json`)

### How To Test

- Check out this branch locally and run `npm ci`
- Take a look at vulnerable dependencies and run `npm list <dependency>` for each
- Verify each previously vulnerable dependency is now outside of the vulnerable range

### To Do
